src/openrct2/peep/peep.c

```diff
@@ -9820,13 +9820,27 @@ static sint32 guest_path_find_entering_park(rct_peep *peep, rct_map_element *map
  *  rct2: 0x0069536C
  */
 static sint32 guest_path_find_leaving_park(rct_peep *peep, rct_map_element *map_element, uint8 edges){
-	rct2_peep_spawn* peepSpawn = &gPeepSpawns[0];
-	// Peeps for whatever reason return to their original spawn point
-	// this in future should look for the nearest.
-	if (peep->sprite_index & 1 && gPeepSpawns[1].x != PEEP_SPAWN_UNDEFINED) {
-		peepSpawn++;
+	uint8 chosenSpawn = 0xFF;
```

Comment:
I'd like to see a comment here. Something like `// Send peeps to the nearest spawn point`.